gateway/utils/graphql/mutation.go

```diff
@@ -32,6 +32,7 @@ func (graph *Module) generateAllReq(ctx context.Context, field *ast.Field, token
 			}
 			result.Type = string(utils.Delete)
 			result.Col = col
+			result.DBAlias, _ = graph.GetDBAlias(field)
```

Comment:
Instead of doing this, why dont you just pass the dbAlias as a parameter in this function